packages/airnode-abi/package.json

```
@@ -10,8 +10,8 @@
   ],
   "scripts": {
     "build": "yarn run clean && yarn run compile",
-    "clean": "rimraf -rf ./dist *.tgz",
-    "compile": "tsc -p tsconfig.build.json",
+    "clean": "rm -f *.tsbuildinfo && rimraf -rf ./dist *.tgz",
```

I think this will break on windows?

```suggestion
    "clean": "rimraf -rf ./dist *.tgz *.tsbuildinfo",
```

---

Will do. Btw. I am not sure we even support windows development. We [recommend using WSL2](https://github.com/api3dao/airnode/blob/master/README.md?plain=1#L63) for a while now.